docs/distributed/REBALANCE.md

```diff
@@ -0,0 +1,74 @@
+# Pool rebalance
+
+## Introduction
+A MinIO distributed setup consists of one or more server pools. MinIO distributes objects uploaded among these pools proportional to their available drive space. A good proportion of objects uploaded will land on the recently added pools. Over time, objects would be uniformly distributed across all the pools.
+
+Certain workloads may require that objects are distributed uniformly across all pools as much as possible, at all times. To support this, MinIO needs to redistribute objects from existing pools to the newly added pools. This is now possible with the Pool rebalance feature.
+
+## Mechanism
+Pool rebalance operates with the goal of converging percentage used space across all server pools. This is done by moving objects from pools that have more objects to those which have lesser, usually because they were recently added.
+
+Let's take an example of a MinIO setup with a single server pool with 300TiB capacity. Over time, let's say its usage reached 200TiB. Now, say a new pool of 300TiB is added. 
+
+```
+   | Pool ID| Used space | Capacity |
+   |--------|------------|----------|
+   | pool-1 |  200 TiB   |  300 TiB |
+   | pool-2 |    0 TiB   |  300 TiB |
+   |--------|------------|----------|
+                         |  600 TiB |
+```
+
+After rebalance, assuming no new objects are uploaded meanwhile, we will have,
+
+```
+   | Pool ID| Used space | Capacity |
+   |--------|------------|----------|
+   | pool-1 |  100 TiB   |  300 TiB |
+   | pool-2 |  100 TiB   |  300 TiB |
+```
+
+## Usage
+Pool rebalance can be managed using `mc`. There are 3 subcommands, namely `rebalance-start`, `rebalance-status` and `rebalance-stop`.
+
```

Comment:
The simple example explains the concept, but not the details. What happens with >2 pools? Different capacities?

Either make the example more complex or have a separate, more complex example.